spec/unit/mutant/ast/named_children_spec.rb

```diff
@@ -42,8 +42,18 @@
       end
 
       describe '#bar' do
-        it 'returns named child bar' do
-          expect(instance.bar).to be(node_bar)
+        context 'when node is present' do
+          it 'returns named child bar' do
+            expect(instance.bar).to be(node_bar)
+          end
+        end
+
+        context 'when node is NOT present' do
```

Comment:
I reflect how I typically use to communicate negations in a development context, to make sure these are not missed. 

Maybe it something specific to the fact I'm not a native speaker, and fear to miss writing or reading negations wrong.

Also I realize how I'm inconsistent in the existing specs. I'm fine to do a followup to bring it in sync as soon we decided about the best strategy.